nexus/src/app/background/init.rs

```diff
@@ -430,17 +434,16 @@ impl BackgroundTasks {
 
         // Background task: abandoned VMM reaping
         let task_abandoned_vmm_reaper = driver.register(
-        String::from("abandoned_vmm_reaper"),
```

Comment:
Here's an example where for some reason the long string caused rustfmt to format the whole statement wrong.